owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -58,13 +60,21 @@ public class WhatsNewActivity extends FragmentActivity implements ViewPager.OnPa
     private ProgressIndicator mProgress;
     private ViewPager mPager;
 
+    private WhatsNewActivityBinding binding_activity;
+    private static WhatsNewElementBinding binding_element;
```

Comment:
BTW, this variable is only used in the inner `FeatureFragment`, so maybe it's better if you move it there (declaration and initialization) since it belongs there. That way, you can save that unnecessary `static` as well 👍